pymemcache/client/base.py

```diff
@@ -660,6 +660,38 @@ def stats(self, *args):
 
         return result
 
+    def slabs(self, *args):
+        """
+        The memcached "slabs" command.
+
+        The returned keys depend on what the "slabs" command returns.
+        A best effort is made to convert values to appropriate Python
+        types, defaulting to strings when a conversion cannot be made.
+
+        Args:
+          *arg: extra string arguments to the "stats" command. See the
+                memcached protocol documentation for more information.
+
+        Returns:
+          If no exception is raised, always returns True.
```

Comment:
Which kind of exceptions was raised?
I guess you need to precise exceptions types etc...